src/c++/library/grpc_client.cc

```diff
@@ -349,6 +350,25 @@ InferResultGrpc::IsNullResponse(bool* is_null_response) const
   return Error::Success;
 }
 
+Error
+InferResultGrpc::Output(std::string& output) const
+{
+  // Only supports LLM outputs with name 'text_output' currently
+  if (output_name_to_buffer_map_.find("text_output") ==
+      output_name_to_buffer_map_.end()) {
+    return Error::Success;
```

Comment:
Returning success without modifying the output variable allows existing workflows to work.